src/main/java/pl/betoncraft/betonquest/compatibility/citizens/CitizensParticle.java

```diff
@@ -75,22 +82,81 @@ public void run() {
 		enabled = true;
 	}
 
+
 	@Override
 	public void run() {
-		for (NPC npc : npcs) {
-			if (npc == null) {
-				continue;
-			}
-			Entity e = npc.getEntity();
-			if (e == null) {
-				continue;
-			}
-			Location loc = e.getLocation().clone();
-			loc.setPitch(-90);
-			Compatibility.getEffectManager().start(name, section, loc);
-		}
+	    // Bukkit.getLogger().info("Particle: RUN:89: Online Players: " + Bukkit.getOnlinePlayers().size() + "NPC:" + npcs.size());
```

Comment:
Could you clean up those comments?
And while you're at it, please replace all tabs with 4 spaces each, it will make the code look good on GitHub (it has 8-spaces long tabs by default).